lib/src/component2_suggestors/component2_utilities.dart

```diff
@@ -141,7 +141,6 @@ bool fullyUpgradableToComponent2(ClassDeclaration classNode) {
 /// * Generic parameters on component class
 /// * `@AbstractProps` in the same file
 bool canBeExtendedFrom(ClassDeclaration classNode) {
-  var a = classNode.typeParameters;
```

Comment:
This was just an unused variable that was popping up in my lints. It doesn't relate to the rest of the PR at all.